app/components/recirculation-prefooter.js

```diff
@@ -123,11 +123,13 @@ export default Component.extend(
 						}));
 
 					run.scheduleOnce('afterRender', () => {
```

Comment:
why do we even have this afterRender here?

Reply:
I guess this is because we need to wait for recirc items to render to setup tracking